.github/workflows/linux.yml

```diff
@@ -113,6 +113,40 @@ jobs:
           name: release-binary
           path: target.tar.gz
 
+  build-flatpak:
```

Comment:
A entire extra build on the CI for every change seems too much of a price to pay for this feature. Probably we need a bigger conversation about when / how this should be built and how it should be distributed. Does it replace the existing Linux package for instance? Right now, Servo it not a usable application....it's more like a demonstration of the current state of the engine. I'm curious what the target audience of the flatpak package is?

Reply:
For your first point, yes. I think it's too high price, and probably even nightly build is questionable to have.

I did it to be sure it's working outside my machine mostly.

I'm well aware it's more about the demonstration ( // I think I have Servo screenshot running on Linux OnePlus 6 in Flatpak somewhere from 2021).

The target audience was meant to be developers considering their web-app to be packaged with Servo, which I think could be use of it (offline or online).

Reply:
This should likely move to the nightly build workflow as we don't really want to be building servo twice for every run of the main workflow.